test/models/network.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+var expect = require("chai").expect;
+
+var Chan = require("../../src/models/chan");
+var Network = require("../../src/models/network");
+
+describe("Network", function() {
+	describe("#export()", function() {
+
+		it("should produce an valid object", function() {
+			var network = new Network({name: "networkName"});
+			network.channels.push(new Chan({name: "#thelounge"}));
+			network.channels.push(new Chan({name: "&foobar"}));
+
+			expect(network.export()).to.deep.equal({
```

Comment:
Yes, on purpose. If changing the behavior or `.export()`, it must be a conscious choice and therefore the test(s) should be updated :-)